sbnci/PlottingScripts/CompareDataDistributions.C

```diff
@@ -326,13 +326,11 @@ void CompareDataDistributions(TString gCurVersion="v07_06_00", TString gRefVersi
               hRef = (TH1D*)RefFile->Get(Form("%s/%s", s_branch_names.Data(), s_histo_names.Data()));
               hCur = (TH1D*)CurFile->Get(Form("%s/%s", s_branch_names.Data(), s_histo_names.Data()));
 
-              if (hRef->Integral() > 0 && hCur->Integral() > 0) {
+              if (hRef->Integral() > 0 && hCur->Integral() > 0 && useNormalised)
```

Comment:
Just a minor change, mostly philosophical on the order of checks in the `if` conditions. 
In here I'd move the check on useNormalised first
`if (useNormalised && hRef->Integral() > 0 && hCur->Integral() > 0)`
so if the normalization is not required the other check are just skipped.